src/app/application.cpp

```diff
@@ -491,13 +491,16 @@ int Application::exec(const QStringList &params)
 #ifndef DISABLE_WEBUI
     Preferences* const pref = Preferences::instance();
     // Display some information to the user
-    std::cout << std::endl << "******** " << qPrintable(tr("Information")) << " ********" << std::endl;
-    std::cout << qPrintable(tr("To control qBittorrent, access the Web UI at http://localhost:%1").arg(QString::number(pref->getWebUiPort()))) << std::endl;
-    std::cout << qPrintable(tr("The Web UI administrator user name is: %1").arg(pref->getWebUiUsername())) << std::endl;
+    const QString mesg = QString("\n******** %1 ********\n").arg(tr("Information"))
```

Comment:
I would use multiple `printf()` call instead of string concatenations (e.g. one call per output line).

Reply:
I suspect the current way (concatenations) is cheaper than multiple printf calls.